src/SolverInterface/lpqp_to_conic.jl

```diff
@@ -7,6 +7,7 @@
 
 type LPQPtoConicBridge <: AbstractConicModel
     lpqpmodel::AbstractLinearQuadraticModel
+    qcp::Bool
```

Comment:
Oops, I forgot to change this. I initially wanted to use `Asoc` and `Arsoc` but they would be `nothing` when there are only SOC variables and no SOC constraint. This is fixed in the commit I just pushed.